utility.py

```diff
@@ -0,0 +1,17 @@
+from datetime import date, datetime
+import psycopg2
+import os
+
+def instantiates_db_connection():
```

Comment:
Nit: `instantiates` implies a condition.
```suggestion
def instantiate_db_connection():
```